.github/workflows/env-setup-deploy-redis.yml

```diff
@@ -0,0 +1,40 @@
+name: Env Setup - Deploy Redis in Openshift
+run-name: Env Setup - Deploy Redis in Openshift from ${{ inputs.gitRef }} to ${{ inputs.environment }}
+
+concurrency: redis-setup
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      gitRef:
+        description: "Build Ref"
+        required: true
+        default: ""
+
+jobs:
+  deployRedis:
+    name: Deploy Redis in Openshift
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Init Redis
+        working-directory: "./devops/"
+        run: |
+          make init-redis NAMESPACE=${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
```

Comment:
https://github.com/bcgov/SIMS/pull/3136/files#r1571158531

Reply:
As we talked, since we are setting the `NAMESPACE` in `env` the command can be invoked as
```
make init-redis
```